tuist/tuist
# Add default known regions in project options

Resolves discussion : https://github.com/tuist/tuist/discussions/4846

### Short description 📝

Add `defaultKnownRegions` in project options
1. User can set custom `defaultKnownRegions` in project options, for example `["en-GB", "Base"]`
2. If no option is set, it defaults to `["en", "Base"]`

### How to test the changes locally 🧐

> Include a set of steps for the reviewer to test the changes locally (see [the documentation](https://docs.tuist.io/contributors/testing-strategy) for reference).

- Unit tests : `./fourier test tuist unit`
- `Tests/TuistGeneratorTests/Generator/ProjectDescriptorGeneratorTests.swift`

### Checklist ✅
